h/templates/client/annotation.html

```diff
@@ -97,6 +97,13 @@
 </div>
 <!-- / Tags -->
 
+<div class="annotation-section small" ng-if="vm.editing">
+  <p ng-show="level.text == 'Only Me'">
```

Comment:
In other words, this scope of the help text should be tied to the current level of the Privacy directive. The way things are currently written, does it make sense that current state of the privacy directive is so separate from the annotation directive?

Should I try binding a property from the annotation controller to the isolated scope of the privacy directive?

Example:
```
scope:
    level: '='
```

Or is this also a form of 'reaching up' that I should avoid?